### ISSUE TITLE
Sorting type column headers 'Source type, target type' are not accessible using keyboard: A11y_WinForm.NETRuntime_Keyboard.

### ISSUE THREAD

**Issue by shindepra** [dotnet/winforms#13760]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."

### GitHub Tags:
#A11ySev2; #A11yTCS; #WCAG2.1.1; #Win32; #DesktopApp; #Win11; #WinForms .NET Runtime; #BM_WinForms.NETRuntime_Win32_June2025; #Keyboard; #A11yMAS

### Environment Details:
Microsoft Windows 11 Enterprise (24H2) Build on 26100.4061  

### Repro Steps:

1. Open SQL Server Migration Assistant for oracle.  
2. Splash screen appears and SSMA is launched 
3. Navigate to the 'Tools' menu and select it. 
4. Navigate to 'Project settings menu item and select it. 
5.  Navigate to "Type Mapping" button select it. 
6. Navigate all controls under "Type Mapping" button. 
7. Verify whether sort functionality column headers are accessible using keyboard or not. 

### Actual Result:
Sorting type column headers 'Source type, target type' are not accessible using keyboard. 

### Expected Result:
Sorting type column headers 'Source type, target type' should be accessible using keyboard. 

### User Empact:
Keyboard users will be affected as they miss using the feature and the functionality of the controls if the keyboard focus doesn't land on the interactive elements on the page. Keyboard users cannot interact and miss accessing the content. 

Sample App used:-

[[[[RadioButtonFocusDemo.zip](https://github.com/user-attachments/files/21483118/RadioButtonFocusDemo.zip)](https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-kkumar2_microsoft_com/ETnjjCF0jsZFiF_P9fDDmyIBlb80rXpmeIDu2or6768kVg?e=M4uY2O)](https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-kkumar2_microsoft_com/ETnjjCF0jsZFiF_P9fDDmyIBlb80rXpmeIDu2or6768kVg?e=cCZLff)](https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-kkumar2_microsoft_com/ETnjjCF0jsZFiF_P9fDDmyIBlb80rXpmeIDu2or6768kVg?e=wBZc6R)

### Attachment:

<img width="843" height="503" alt="Image" src="https://github.com/user-attachments/assets/2cd76d13-25c4-4e36-8aac-9b2a610a3e1e" />

https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-kkumar2_microsoft_com/EQScqHS_FkhFmP-HT-YhcbcBMlvzKR4Hn_FM5I5f6IOoAw?nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJPbmVEcml2ZUZvckJ1c2luZXNzIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXciLCJyZWZlcnJhbFZpZXciOiJNeUZpbGVzTGlua0NvcHkifX0&e=16rVi1

**Comment by LeafShi1** [comment]
The default keyboard behavior of DataGridView only supports navigation between cells, and the row header is not a focusable element, please refer to the [official documentation](https://learn.microsoft.com/en-us/dotnet/desktop/winforms/controls/default-keyboard-and-mouse-handling-in-the-windows-forms-datagridview-control)
